### DISCUSSION TITLE
alt+shift+arrow, how to set distance or length between base element and new element

### DISCUSSION THREAD

**Discussion by sdykae** [jgraph/drawio#3781] [General]
Question, how to configure default length when using the shortcut keys to create new elements based on selected one

**Comment by alderg** [comment]
Use defaultEdgeLength = 200 in the Configuration or use [this link](https://app.diagrams.net/#_CONFIG_UzV3UjUySklNSyzNKXFNSU/1Sc1LL8kAiqkaOxoZGKiauwAA). See https://www.drawio.com/doc/faq/configure-diagram-editor

```
{
  "defaultEdgeLength": 200
}
```
